src/assets/css/main.scss

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: 'Rubik';
+  src: url(~/assets/fonts/Rubik-Regular.woff) format('woff');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Rubik';
+  src: url(~/assets/fonts/Rubik-Bold.woff) format('woff');
+  font-weight: 700;
+  font-style: normal;
+}
+
+* {
+  color: $base-text-color;
+}
+
+h1 {
+  font-family: 'Rubik';
+  font-style: normal;
+  font-weight: 700;
+  font-size: var(--main-heading-textsize);
+  line-height: 118.75%;
+  margin: 0;
+
+  text-transform: uppercase;
+}
+
+h2 {
+  font-family: 'Rubik';
+  font-style: normal;
+  font-weight: 700;
+  font-size: var(--section-heading-textsize);
+  line-height: 118.75%;
+  margin: 0;
+  text-transform: uppercase;
+  text-align: center;
+
+  &.smaller {
+    font-size: var(--section-heading-textsize-small);
+    line-height: 117.857%;
+  }
+}
+
+li {
+  list-style-type: disc;
+}
+
+h3 {
+  font-family: 'Rubik';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 120.28%;
```

Comment:
Where does this value come from?

Reply:
Taken from the Styleguide. Looks good on all platforms.